tests/publiccloud/registration.pm

```diff
@@ -36,7 +37,11 @@ sub run {
 
 sub cleanup {
     my ($self) = @_;
-    $self->{instance}->upload_log('/var/log/cloudregister', log_name => $autotest::current_test->{name} . '-cloudregister.log');
+    my @logs = ('/var/log/cloudregister', '/etc/hosts', '/var/log/zypper.log', '/etc/zypp/credentials.d/SCCcredentials');
```

Comment:
I would rename this variable since these are not strictly logs.  Maybe `@files` is more appropiate here?